### ISSUE TITLE
no file

### ISSUE THREAD

**Issue by puks** [jamesla/backgroundvideo#6]
Hello! great work on this. However, I cannot find the file after the app recorded a video. theres no folder under my app's name even on android/data/app_name/cache or android/data/app_name/tmp for that natter. pleas help. how can we get the file after the recording via javascript. thanks.

**Comment by kmturley** [comment]
Yeah it doesn't seem to be writing videos for me either. I've tried changing the following things:

plugin.xml
line 44 - lines 47

```
<uses-permission android:name="android.permission.CAMERA" />
<uses-permission android:name="android.permission.RECORD_AUDIO" />
<uses-permission android:name="android.permission.RECORD_VIDEO"/>
<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />android:name=&quot;android.permission.WRITE_EXTERNAL_STORAGE&quot; /&gt;
```

BackgroundVideo.java
line 33 - 41

```
FILE_PATH = Environment.getExternalStorageDirectory().toString() + "/";
File folder = new File(FILE_PATH);
if (!folder.exists()) {
    folder.mkdir();
    Log.d(TAG, "folder.created: " + FILE_PATH);
} else {
    Log.d(TAG, "folder.exists: " + FILE_PATH);
}
```

But i'm getting the following output:

```
folder.exists: /storage/emulated/0/
start Recording: /storage/emulated/0/myvideo.mp4
stop: 
```

And trying to use Android File Transfer or adb pull to get the files, no luck:

```
adb pull /data/data/. /.
```

**Comment by dairejpwalsh** [comment]
I changed the following in BackgoundVideo.java on line 34,

FILE_PATH = "/storage/emulated/0/Pictures/";

Now all video's saved to Pictures folder in Gallery

**Comment by jhayward** [comment]
The latest version now returns the file path on stop - does this help?
Do you still have the issue when running the demo?

**Comment by quantodaniel** [comment]
Hi! I'm running the demo on iOS and the stop function is returning `undefined`. Any idea why?

![1](https://cloud.githubusercontent.com/assets/2720511/10126709/a2ffe55e-656d-11e5-98e0-30630ff39096.PNG)![2](https://cloud.githubusercontent.com/assets/2720511/10126714/ad304e6a-656d-11e5-9b16-00a4ffe6ab0d.PNG)

**Comment by quantodaniel** [comment]
I guess the problem is with `cordova plugin add io.iclue.backgroundvideo`.. downloading from github works fine!

**Comment by crystaltong** [comment]
Hello! The ios version isn't going saving to my library either. Any suggestions?

**Comment by jamesla** [comment]
Hi is this still a problem with the newer versions?
